daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md

```diff
@@ -0,0 +1,55 @@
+---
+type: docs
+title: "Actors activation performance"
+linkTitle: "Actors activation performance"
+weight: 10000
+description: ""
+---
+This article provides service invocation API performance benchmarks and resource utilization for Actor in Dapr on Kubernetes. 
+
+## System overview
+
+For applications using Actors in Dapr there are two aspects to be considered. First, is the routing of actor invocations handled by Dapr sidecar. Second, is the Actor runtime that is implemented and handled on the application side and depends on the SDK. For now, the performance tests are using the Java SDK to provide an Actor runtime in the application.
```

Comment:
lets be consistent with capitalizing (actor or Actor)

Reply:
Yes. You should use lowercase "actor" and "actors". This is not a name.